code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/azure/AzureRESTClient.java

```diff
@@ -237,16 +237,22 @@ public void refreshAccessToken() {
 
     public AzureAccountInfo jsonToAzureAccountInfo(Map<String, Object> jsonData) {
         String objectId = ObjectUtils.toString(jsonData.get("objectId"));
+        String objectType = ObjectUtils.toString(jsonData.get("objectType"));
```

Comment:
ObjectUtils.toString() will return an empty string if `obj` is `null`.  Is this really what you want to do?